test/scales/modifiedLogScaleTests.ts

```diff
@@ -2,187 +2,241 @@
 
 describe("Scales", () => {
   describe("Modified Log Scale", () => {
-    let scale: Plottable.Scales.ModifiedLog;
-    let base = 10;
-    let epsilon = 0.00001;
-    beforeEach(() => {
-      scale = new Plottable.Scales.ModifiedLog(base);
-    });
 
-    it("is an increasing, continuous function that can go negative", () => {
-      d3.range(-base * 2, base * 2, base / 20).forEach((x: number) => {
-        // increasing
-        assert.operator(scale.scale(x - epsilon), "<", scale.scale(x));
-        assert.operator(scale.scale(x), "<", scale.scale(x + epsilon));
-        // continuous
-        assert.closeTo(scale.scale(x - epsilon), scale.scale(x), epsilon);
-        assert.closeTo(scale.scale(x), scale.scale(x + epsilon), epsilon);
-      });
-      assert.closeTo(scale.scale(0), 0, epsilon);
-    });
+    describe("Basic Usage", () => {
+      let scale: Plottable.Scales.ModifiedLog;
+      let base = 10;
+      let epsilon = 0.00001;
```

Comment:
Hmm, while the old tests do this, currently the `epsilon` variable holds 2 semantics. It is, for one, the delta for choosing close data values, and it is also the error to use for close to.  Should we split this up into two variables?